Gemfile

```diff
@@ -16,14 +16,14 @@ gem "sass" #3.2.12
 # gem "middleman-syntax"
 
 #deployment
-gem "middleman-deploy" #, "~> 2.0.0.pre.alpha"
+gem "middleman-deploy", "~> 2.0.0.pre.alpha"
```

Comment:
Based on https://github.com/karlfreeman/middleman-deploy/issues/132#issuecomment-337524002 doesn't work without for me